docs/synchronizations-faq.md

```diff
@@ -1,16 +1,79 @@
-# Synchronization FAQ
-##
+# Synchronization
+## Synchronization of Users, Teams, Permissions and Repositories
 
-Synchronizations allow you to keep packages, teams, their members, and access permissions in sync with your code hosting platform, e.g. GitHub, GitLab or Bitbucket.
+To simplify the initial setup and maintenance of a Private Packagist account, we offer optional synchronization for GitHub organizations, 
+Bitbucket workspaces and GitLab groups. This synchronization keeps teams, their members, and access permissions in sync. 
+So you only need to manage users and permissions in a single place.
 
-## What happens if you promote a synchronization to a primary synchronization?
+Apart from synchronizing teams, users, and permissions, setting up the integration will simplify the addition of new packages 
+to your Composer repository. When you create a **new repository** on GitHub, Bitbucket, or GitLab, it will be **added as a 
+Composer package automatically** if it contains a _composer.json_ file. If you’d like to add existing repositories as packages, 
+you can do so with the click of a button on the Packages tab in your organization.
+
+**Update web hooks** notifying Packagist of new code pushes will be installed in the background, and **credentials to access
+the package** will be **configured automatically**.
+
+## Getting started
+
+You can log into Private Packagist using an account on GitHub.com, Bitbucket.org, or GitLab.com with the relevant [OAuth integration](integration-github-bitbucket-gitlab.md)
+If you already have an account you can connect these services to your existing account on your [profile page](https://packagist.com/profile).
+
+You can then either create a new organization directly synchronized with a GitHub organization, Bitbucket workspace or a GitLab group,
+or you can enable synchronization for an existing Organization in _Settings_.
+
+![Synchronization](/Resources/public/img/docs/features/Sync-20241218.png)
+
+## Supported providers 
+
+### GitHub
+* Synchronization:
+    * Keeps teams, their members, and access permissions in sync with your GitHub organization
+* Code Credentials: GitHub App or GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
```

Comment:
I think this (Code Creds/Webhooks) is also relevant on this page, but kept it in place on the OAuth Integrations page too. So it's duplicated, but I think we should mention it on both locations. Thoughts?

Reply:
Makes sense to have it in both place but I would also keep the `Synchronization:` section on the integration page. This makes it easier to compare which features are available for which integration